rustls/src/crypto/ring/hmac.rs

```diff
@@ -5,7 +5,7 @@ use crate::crypto;
 
 pub(crate) static HMAC_SHA256: Hmac = Hmac(&ringlike::hmac::HMAC_SHA256);
 pub(crate) static HMAC_SHA384: Hmac = Hmac(&ringlike::hmac::HMAC_SHA384);
-#[cfg(all(test, feature = "tls12"))]
+#[allow(dead_code)] // only for TLS1.2 prf test
```

Comment:
If this remains test-only can we at least still guard it with `cfg(test)`?

Reply:
have done that